_partials/_cloud-mst-comparison.mdx

```diff
@@ -0,0 +1,11 @@
+<highlight
+  type="cloud"
+  header="Timescale Cloud or Managed Service for TimescaleDB?"
+  button="Go to Timescale Cloud"
+>
+  Choosing between Timescale Cloud and Managed Service for TimescaleDB? On AWS,
+  we recommend Timescale Cloud, which we designed specifically for TimescaleDB.
+  On Azure, GCP, and some less common regions of AWS, we recommend our partnered
+  service, Managed Service for TimescaleDB. [See the available Timescale Cloud
+  regions.](/cloud/latest/#available-in-multiple-aws-regions)
```

Comment:
I'm not 100% convinced that this is the right link. It feels a little jarring to be like "here's how to choose" then straight into "Cloud regions". Maybe we need to have a page that goes into a little more detail (I mean, there are significant feature differences between the two as well, it's not just regions)? Or link to MST regions as well? I'm not sure ...

Reply:
Hi @charislam  and @Loquacity  Here's what Alex shared with me 
<img width="988" alt="Screenshot 2022-08-29 at 10 34 23 AM" src="https://user-images.githubusercontent.com/37059749/187126871-978dc48f-3f9d-44c0-8168-4bf733f2aa5b.png">

Reply:
Yes I agree it might be a bit abrupt. The purpose is to simplify the "how to choose." Clouds and regions is the major factor and is how our signup wizard works as well. But I'm going to fine-tune this a bit to make it less jarring!